app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+from os import (
+    makedirs,
+    path
+)
+
+from shutil import move
+
+
+class MoveCommandSyntaxError(Exception):
+    pass
+
+
+def move_file(command: str) -> None:
+    if not command.startswith("mv"):
+        raise MoveCommandSyntaxError("Move command not found")
+
+    command = command.split()
+
+    if len(command) != 3:
+        raise MoveCommandSyntaxError(
+            "Invalid move command syntax.\n"
+            "For a successful moving of file "
+            "the command should look like this: "
+            "mv file.txt some_dir/new_file.txt"
+        )
+
+    mv_command, file_to_move, new_file_path = command
+    is_consist_directory = "/" in command[-1]
```

Comment:
Is ur code cross-platform?) For example, will it work on win with \ ? Use os.path.split() instead